lib/Cleantalk/ApbctWP/LinkConstructor.php

```diff
@@ -142,6 +142,14 @@ class LinkConstructor extends \Cleantalk\Common\LinkConstructor
             'utm_medium' => 'admin_bar',
             'utm_content' => 'renew_admin_bar_spbct',
         ),
+        'exclusion_by_form_signs' => array(
+            'utm_id' => '',
+            'utm_term' => '',
+            'utm_source' => 'apbct_hint_exclusions__form_signs',
```

Comment:
The utm_source should show what the common source that had called the link click. 

The utm_medium is used to specify this place in more details

Something like this:
..
            'utm_id' => '',
            'utm_term' => '',
            'utm_source' => 'admin_panel',  //WP place
            'utm_medium' => 'settings', // WP place details
            'utm_content' => 'apbct_hint_exclusions__form_signs', //the link destination
...